Dalamud/Data/DataManager.cs

```diff
@@ -185,15 +185,17 @@ public bool FileExists(string path)
     /// </summary>
     /// <param name="iconId">The icon ID.</param>
     /// <returns>The <see cref="TexFile"/> containing the icon.</returns>
-    /// TODO(v9): remove in api9 in favor of GetIcon(uint iconId, bool highResolution)
+    [Obsolete("Use ITextureProvider instead")]
     public TexFile? GetIcon(uint iconId) 
         => this.GetIcon(this.Language, iconId, false);
```

Comment:
Things that return `TexFile` are technically being removed without a replacement, no? Pedantics, but might be worth calling that out explicitly or mentioning that plugins that need raw Tex files are now on their own (though this could be in the dev patch notes as well).

Reply:
Added a function to allow path lookups for icons